### PR TITLE
Update ice prison (and lightning orb sound)

### LINKED ISSUE #612: add ice prison and lightning orb sound
add the sound for when you throw ice prison and when it lands, for when you throw lorb

### PR DESCRIPTION
**Pull request by StudentAlleg**
Add sounds on throw and travel from ice prison and lightning orb
Change ice prison to decay from top to bottom instead of all at once. The speed is now controlled by the variance variable

Fixes #612

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested my changes.

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/IcePrison.java [hunk 1]
```diff
@@ -35,6 +39,8 @@ public class IcePrison extends Skill implements InteractSkill, CooldownSkill, Li
     private double durationIncreasePerLevel;
     private double speed;
```
Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/IcePrison.java:41]:
```suggestion
```

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/IcePrison.java [hunk 2]
```diff
@@ -104,13 +110,22 @@ private void handleIcePrisonCollision(ThrowableItem throwableItem) {
...
             }
         }
     }
 
+    @UpdateEvent
+    public void doSound() {
+        for (ThrowableItem throwableItem : championsManager.getThrowables().getThrowablesOfName(getName())) {
+            throwableItem.getLastLocation().getWorld().playSound(throwableItem.getLastLocation(), Sound.BLOCK_LAVA_EXTINGUISH, 0.6f, 1.6f);
+        }
+    }
```
[the remark this answers is not in the document]
Comment (StudentAlleg) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/IcePrison.java:128]: Makes sense

[the remark this answers is not in the document]
Comment (StudentAlleg) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/IcePrison.java]: implemented

[the remark this answers is not in the document]
Comment (Mykindos) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/IcePrison.java]: good stuff

**Pull request merged**